dataland-frontend/src/utils/DataPoint.ts

```diff
@@ -12,6 +18,11 @@ export interface BaseDataPoint<T> {
   dataSource?: BaseDocumentReference | null;
 }
 
+export interface ListOfBaseDataPointsFormField {
```

Comment:
This connects to the ability of the backend to receive more than one document, correct?

Reply:
There is no longer this type. Was removed after refactoring

Reply:
ok